backend/dataall/modules/dataset_sharing/services/share_object_service.py

```diff
@@ -533,13 +533,29 @@ def attach_dataset_table_read_permission(session, share):
             session, share.shareUri, ShareableType.Table.value, [ShareItemStatus.Share_Succeeded.value]
         )
         for table in share_table_items:
```

Comment:
Why the TODO?

Reply:
I have some thoughts about this case and how to deal with it, but i think, we better discuss it first. Right now it's not necessary, so I decided to wait.